themes/sandal/less/search.less

```diff
@@ -274,7 +301,7 @@
     .filters-term {
       color: #000;
     }
-    a:hover {
+    a:hover {
```

Comment:
I'm not sure what's going on here -- I noticed that this PR removed some whitespace that it didn't need to remove, so I re-added the whitespace... but now it's still showing up as a diff. I wonder if there was somehow some strange non-printable character here instead of a regular space, but I don't know how or why that would have happened.

Reply:
This was an non-breaking space. My IDE showed it like this [NBSP]. I wanted to replace it with a normal space but somehow only deleted it. I think now it is correct.